runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -81,7 +81,7 @@
     /**
      * The stream view this object is backed by.
      */
-    private final ISMRStream smrStream;
+    private final StreamViewSMRAdapter smrStream;
```

Comment:
Why change from interface to concrete implementation?

Reply:
Because I am using StreamViewSMRAdapter's streamName getter to get the underlying stream.

Reply:
This is not needed if you expose `IStream::getName`.

I think we should just use the `ISMRStream` interface.